src/app.js

```diff
@@ -80,9 +82,10 @@ function startApp(d2) {
 // can use it to access the api, translations etc.
 getManifest('./manifest.webapp')
     .then(manifest => {
-        const baseUrl = process.env.NODE_ENV === 'production' ? manifest.getBaseUrl() : dhisDevConfig.baseUrl;
+        const baseUrl = process.env.NODE_ENV === 'production' ?
+            manifest.getBaseUrl() : dhisDevConfig.baseUrl;
+        config.siteUrl = baseUrl;
```

Comment:
I was also perplexed.  It turns out `config.siteUrl` is used interally by d2 and must be set to `base/api/VERSION`. So I needed to create another key to point to the main site, thus the name.